www/app.css

```diff
@@ -0,0 +1,6 @@
+.content-wrapper {
+  background-color: #fff;
```

Comment:
Is that a standard way of writing a hex colour? Should it be `#ffffff`?

Reply:
I didn't write this but it is an accepted way of writing white. See [short notation](https://www.w3.org/TR/css-color-3/#numerical) in the W3C recommendations.